src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -610,6 +613,10 @@ export class ResultsViewPageStore {
         return getMolecularProfiles(this.urlWrapper.query);
     }
 
+    public handleTabChange(id: string, replace?: boolean) {
+        this.urlWrapper.updateURL({}, `results/${id}`, false, replace);
+    }
```

Comment:
moved from `ResultsViewPage.tsx` to `ResultsViewPageStore.ts` to reduce redundancy.

Reply:
where is the redundancy?

Reply:
Download tab cannot use a private function from `ResultsViewPage.tsx`, so this function need to be copied into `ResultsViewPageStore.ts`, so I just moved it to `ResultsViewPageStore.ts`.